cmd/version.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var Version = "v0.9.2"
+var Version = "dev"
```

Comment:
I think this should be uppercase `VERSION` if you are referring `main.VERSION={{.Tag}}`